waggle-dance-rest/src/test/java/com/hotels/bdp/waggledance/rest/service/DelegatingFederationServiceTest.java

```diff
@@ -35,9 +35,6 @@
 @RunWith(MockitoJUnitRunner.class)
 public class DelegatingFederationServiceTest {
 
-  private static final String FEDERATED_METASTORE1_URI = "thrift://federatedMetaStore1:9083";
```

Comment:
why is the test changed, the code isn't?

Reply:
I think you added this commit. Not sure why were those asserts removed but I have added them back.

Reply:
I take back my words. It was removed because of unnecessary stubbing error being thrown while running tests.

```
Following stubbings are unnecessary (click to navigate to relevant line of code):
  1. -> at com.hotels.bdp.waggledance.rest.service.DelegatingFederationServiceTest.init(DelegatingFederationServiceTest.java:50)
  2. -> at com.hotels.bdp.waggledance.rest.service.DelegatingFederationServiceTest.init(DelegatingFederationServiceTest.java:51)
  3. -> at com.hotels.bdp.waggledance.rest.service.DelegatingFederationServiceTest.init(DelegatingFederationServiceTest.java:53)
Please remove unnecessary stubbings or use 'lenient' strictness. More info: javadoc for UnnecessaryStubbingException class.
	at org.mockito.internal.runners.StrictRunner.run(StrictRunner.java:49)
	at org.mockito.junit.MockitoJUnitRunner.run(MockitoJUnitRunner.java:163)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:252)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:141)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:112)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
```

Reply:
Ah OK, interesting, I'm assuming this is because of the change to the Mockito Runner that is being used and this one is more strict? The other one was deprecated right? If so then these changes are fine.

Reply:
ah yes now I remember, sorry @abhimanyugupta07. 
Yes that's because of the mockito change it nicely shows you what you shouldn't do :)